Sources/ProjectDescription/SchemeDiagnosticsOptions.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+/// Options to configure scheme diagnostics for run and test actions.
+public struct SchemeDiagnosticsOptions: OptionSet, Codable, Hashable {
```

Comment:
This may not scale best for all the diagnostic options 🤐

Any option with further sub-options can't be represented
 e.g. 

![diagnostics](https://user-images.githubusercontent.com/11914919/190224212-6925db1f-c8b1-4e39-bbb8-887ee0f2675b.png)

In the next major release we can consider doing what we did for `.options` and converting it to a `struct` with explicit properties which will be better data model for this.

Reply:
argh, let me go back then 🚀